.travis.yml

```diff
@@ -1,24 +0,0 @@
-language: php
-
-dist: precise
-
-php:
-    - 7.2
-    - 7.3
-    - nightly
```

Comment:
Can't run on `nightly` - do we care?

Reply:
No really if you ask me. Isnt there a 7.4 image available already since release is in 2 weeks from now.

Reply:
Is available now, yes!